.github/workflows/ci-cd.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           cd backend
```

Comment:
Apparently some formatting missing... What's interesting is the added `Push grafana to Dockerhub` step